{"info": {"author": "Tu Yuxiao", "author_email": "754738085@qq.com", "bugtrack_url": null, "classifiers": [], "description": "", "description_content_type": "", "docs_url": null, "download_url": "", "downloads": {"last_day": -1, "last_month": -1, "last_week": -1}, "home_page": "https://github.com/TuYuxiao/face_recognize", "keywords": "face recognize", "license": "MIT License", "maintainer": "", "maintainer_email": "", "name": "face-recognize", "package_url": "https://pypi.org/project/face-recognize/", "platform": "any", "project_url": "https://pypi.org/project/face-recognize/", "project_urls": {"Homepage": "https://github.com/TuYuxiao/face_recognize"}, "release_url": "https://pypi.org/project/face-recognize/0.2.0/", "requires_dist": null, "requires_python": "", "summary": "face recognize with dlib and arcsoft toolkit", "version": "0.2.0", "yanked": false, "html_description": null}, "last_serial": 6547000, "releases": {"0.1.0": [{"comment_text": "", "digests": {"md5": "57f4292d6d8ae56ea4b0e9474a5c3896", "sha256": "2ded20355b29ec2325becda4bd96e07fa06bf0f750bcca4044650d06f42d0c36"}, "downloads": -1, "filename": "face_recognize-0.1.0.tar.gz", "has_sig": false, "md5_digest": "57f4292d6d8ae56ea4b0e9474a5c3896", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 16279, "upload_time": "2020-01-28T04:33:45", "upload_time_iso_8601": "2020-01-28T04:33:45.719811Z", "url": "https://files.pythonhosted.org/packages/f4/1a/65c4dea1bb2472cb226da570bac1d091c2012c73c68f799691d65082bed6/face_recognize-0.1.0.tar.gz", "yanked": false}], "0.2.0": [{"comment_text": "", "digests": {"md5": "d0b7e5fea7b0272a3cc32f74f8eec5c8", "sha256": "d499398094121eb29728476fe47098711c14f24bd0571d53505229d9051884a9"}, "downloads": -1, "filename": "face_recognize-0.2.0.tar.gz", "has_sig": false, "md5_digest": "d0b7e5fea7b0272a3cc32f74f8eec5c8", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 16811, "upload_time": "2020-01-31T04:44:42", "upload_time_iso_8601": "2020-01-31T04:44:42.027704Z", "url": "https://files.pythonhosted.org/packages/c6/3f/997ad84a1c551ee987b86fdaa2f83ad056909a170c5763f40c691eb1c9bb/face_recognize-0.2.0.tar.gz", "yanked": false}]}, "urls": [{"comment_text": "", "digests": {"md5": "d0b7e5fea7b0272a3cc32f74f8eec5c8", "sha256": "d499398094121eb29728476fe47098711c14f24bd0571d53505229d9051884a9"}, "downloads": -1, "filename": "face_recognize-0.2.0.tar.gz", "has_sig": false, "md5_digest": "d0b7e5fea7b0272a3cc32f74f8eec5c8", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 16811, "upload_time": "2020-01-31T04:44:42", "upload_time_iso_8601": "2020-01-31T04:44:42.027704Z", "url": "https://files.pythonhosted.org/packages/c6/3f/997ad84a1c551ee987b86fdaa2f83ad056909a170c5763f40c691eb1c9bb/face_recognize-0.2.0.tar.gz", "yanked": false}], "timestamp": "Fri May  8 00:44:05 2020"}